addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
The anglosaxon feature and Odoo itself is not multicompany aware in terms of localization IMHO it is ok in this way, in case that at least 1 company have it configured the one that need multi-localization will need to make an external feature.